Dockerfile

```diff
@@ -65,17 +61,18 @@ export GOROOT=/usr/local/go && \
 export GOPATH=/tmp/cilium-net-build && \
 export PATH="$GOROOT/bin:/usr/local/clang+llvm/bin:$GOPATH/bin:$PATH" && \
 go get -u github.com/jteeuwen/go-bindata/... && \
+cd /tmp/cilium-net-build/src/github.com/cilium/cilium && \
+export GOROOT=/usr/local/go && \
+export GOPATH=/tmp/cilium-net-build && \
+export PATH="$GOROOT/bin:/usr/local/clang+llvm/bin:$GOPATH/bin:$PATH" && \
 make clean-container all && \
 make PKG_BUILD=1 install && \
 groupadd -f cilium && \
-# cilium-end
-#
+
 apt-get purge --auto-remove -y gcc make bison flex git curl xz-utils ca-certificates && \
 # Needed for system minimal requirements checkers
-apt-get install -y --no-install-recommends libgcc-5-dev binutils && \
```

Comment:
By that do you mean we should cut a version that corresponds to the version being cut for Cilium?